packages/runtime/src/state/modules/components-meta.ts

```diff
@@ -1,15 +1,16 @@
 import { Action, ActionTypes } from '../actions'
 
-export type ComponentIcon =
-  | 'Carousel40'
-  | 'Code40'
-  | 'Countdown40'
-  | 'Cube40'
-  | 'Divider40'
-  | 'Form40'
-  | 'Navigation40'
-  | 'SocialLinks40'
-  | 'Video40'
+export enum ComponentIcon {
```

Comment:
We avoid using TypeScript enums because it's not standard JavaScript. We prefer to use TypeScript in a way were all types can just be compiled away and the JavaScript isn't affected.

https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh

Instead define it like this:

```ts
const ComponentIcon = {
} as const
type ComponentIcon = typeof ComponentIcon[keyof typeof ComponentIcon]
```